recipe/meta.yaml

```diff
@@ -7,25 +7,25 @@ package:
 source:
     fn: wradlib-{{ version }}.tar.gz
     url: https://pypi.python.org/packages/source/w/wradlib/wradlib-{{ version }}.tar.gz
```

Comment:
@kmuehlbauer change this to:

`https://pypi.io/packages/source/w/wradlib/wradlib-{{ version }}.tar.gz`

The old PyPI URLs no longer work. You can be re-direct to the new ones sometimes, but those are long and ugly. We are using the `pypi.io` for now.